common/djangoapps/util/password_policy_validators.py

```diff
@@ -42,12 +44,12 @@ def validate_password_length(value):
     message = _("Invalid Length ({0})")
     code = "length"
 
-    min_length = getattr(settings, 'PASSWORD_MIN_LENGTH', None)
-    max_length = getattr(settings, 'PASSWORD_MAX_LENGTH', None)
+    min_length = PASSWORD_MIN_LENGTH
```

Comment:
I am concerned about this change. The settings are probably specified in another openedx installation. I think Mackenzie Academy open-edx instance uses these settings